types/mailgun-js/mailgun-js-tests.ts

```diff
@@ -76,3 +76,20 @@ const validationResult6: mailgunFactory.validation.ValidateResponse = {
         local_part: "foo"
     }
 };
+
+// Generic requests
+mailgun.get('/samples.mailgun.org/stats', { event: ['sent', 'delivered'] }, (error, body) => {
+    if (error) {
+        console.error(error);
+    } else {
+        console.log(body);
+    }
+});
+
+mailgun.get('/samples.mailgun.org/stats', { event: ['sent', 'delivered'] })
```

Comment:
The code is never executed. You could simply write `const getResponse: Promise<any> = ...`. This would save some lines that don't add any value.